cmd/activator/request_log.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/knative/serving/pkg/activator"
+	"github.com/knative/serving/pkg/apis/serving"
+	pkghttp "github.com/knative/serving/pkg/http"
+	"go.uber.org/zap"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func updateRequestLogFromConfigMap(logger *zap.SugaredLogger, h *pkghttp.RequestLogHandler) func(configMap *corev1.ConfigMap) {
+	return func(configMap *corev1.ConfigMap) {
+		newTemplate := configMap.Data["logging.request-log-template"]
```

Comment:
Can we do this via the schema-based config?

Reply:
It is already done in schema based config. SetTemplate does it one more time, but I didn't want to remove the extra validation from SetTemplate method because SetTemplate can be used without a config map (e.g. in queue-proxy).